packages/pigeon/lib/functional.dart

```diff
@@ -75,3 +75,13 @@ Iterable<T> repeat<T>(T item, int n) sync* {
     yield item;
   }
 }
+
+/// Returns true if any value from [values] evalutes to true for `func(value)`.
```

Comment:
How is this different from https://api.flutter.dev/flutter/dart-core/Iterable/any.html?

Reply:
I thought we had one.  Didn't see it for some reason.  Done.